src/main/scala/cognite/spark/v1/DefaultSource.scala

```diff
@@ -390,6 +410,11 @@ object DefaultSource {
   def saveTracingHeaders[F[_]: Functor: Trace](): F[Seq[(String, String)]] =
     Trace[F].kernel.map(saveTracingHeaders(_))
 
+  def logParameterOrDefault[A](name: String, value: Option[A], default: => A): A = value match {
+    case None => logger.info(s"using default value for ${name}: ${default}"); default
+    case Some(value) => logger.info(s"using supplied value for ${name}: $value"); value
```

Comment:
Here you're re-using the name from the function argument. Different languages have different standards for this stuff, is this acceptable, or even suggested in Scala?

Reply:
generally hiding names can be error-prone, but for very small code scope sounds ok, plus it's only Option unwrapping - meaning doesn't change much and type system would catch most misuse

Don't know if some of scala linters/formatters have configurations for it, at least not the ones currently enabled in this repo

Reply:
Sounds good, thanks for the explanation!